src/app/terminology/terminology-details/terminology-details.component.ts

```diff
@@ -132,7 +154,7 @@ export class TerminologyDetailsComponent implements OnInit {
     this.title.setTitle(`Terminology - ${this.mcTerminology?.label}`);
   }
 
-  validateLabel = data => {
+  validateLabel = (data) => {
```

Comment:
Andrew Tsui suggested that functions written in this property syntax should be changed back to their standard function syntax for consistency, e.g. `validateLabel(data) { }`.

Also, if you happen to know the type of `data` then I'd suggest adding the definition to the parameter as well.